tests/actions/ReportFieldsTest.ts

```diff
@@ -0,0 +1,94 @@
+import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import Onyx from 'react-native-onyx';
+import {generateFieldID} from '@libs/WorkspaceReportFieldsUtils';
+import CONST from '@src/CONST';
+import OnyxUpdateManager from '@src/libs/actions/OnyxUpdateManager';
+import * as Policy from '@src/libs/actions/Policy/Policy';
+import * as ReportFields from '@src/libs/actions/Policy/ReportFields';
+import type {CreateReportFieldArguments} from '@src/libs/actions/Policy/ReportFields';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {PolicyReportField, Policy as PolicyType} from '@src/types/onyx';
+import * as TestHelper from '../utils/TestHelper';
+import type {MockFetch} from '../utils/TestHelper';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+
+OnyxUpdateManager();
+describe('actions/ReportFields', () => {
+    beforeAll(() => {
+        Onyx.init({
+            keys: ONYXKEYS,
+        });
+    });
+
+    beforeEach(() => {
+        global.fetch = TestHelper.getGlobalFetchMock();
+        return Onyx.clear().then(waitForBatchedUpdates);
+    });
+
+    describe('createReportField', () => {
+        it('creates a new text report field of a workspace', async () => {
+            (fetch as MockFetch)?.pause?.();
```

Comment:
```suggestion
            mockFetch?.pause?.();
```

Then you can use it here and in the other places

Reply:
Done - bd5dacd.